storybook/pages/StatusChatInputPage.qml

```diff
@@ -133,6 +125,9 @@ SplitView {
                 onRemovePaymentRequestPreview: (index) => {
                     d.paymentRequestModel.remove(index)
                 }
+                onOpenGifPopupRequest: {
```

Comment:
please use the new handler syntax (it would generate warning on qt6):
```
onOpenGifPopupRequest: (params, cbOnGifSelected, cbOnClose) => {

}
```